lib/api/liskApi.js

```diff
@@ -547,9 +547,9 @@ LiskAPI.prototype.getAccount = function (address, callback) {
  */
 
 LiskAPI.prototype.generateAccount = function (secret, callback) {
-	this.sendRequest('accounts/generatePublicKey', { secret: secret }, function (result) {
-		return callback(result);
-	});
+	var keys = LiskJS.crypto.getPrivateAndPublicKeyFromSecret(secret);
+	callback(keys)
```

Comment:
But all API methods receive the callback as the second parameter and return `this` and I follow that design.